packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -70,41 +76,46 @@ The expandable list adds an expansion panel to every row to reveal more details
 Use an expandable data list when you have more information than will comfortably fit inside a row, or you want to provide a way for advanced users to access information that is not applicable to all users.
 
 ## Tables
-* **Standard table:** Combines the styling of a list view with behaviors that are commonly associated with a table (sometimes referred to as a hybrid list)
-* **Compact table:** Maximizes the amount of data that can be displayed in a small space
 
-### Standard table
+### How to use
+Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
-![standard table](./img/standard-data-table.png)
+### Table capabilities
+Every [compact](#when-to-use-compact-vs-basic-spacing) or [basic](#when-to-use-compact-vs-basic-spacing) table can be extended with these functionalities. 
+
+* **[Expandable data table](#expandable-data-table)**
+* **[Compound expandable table](#compound-expandable-table)**
+* **[Sortable table](#sorting-by-columns)**
+
+### Compact table
```

Comment:
Just noticed that for data list you only have a section for "compact data list" and not for "basic data list", but for tables, you have a section or both. We should definitely do one or the other for both cases... not sure which way we should go though. 

Also, I notice that for the "when to use a compact table" section, you talk about it more as "when to use compact vs basic"
but for the "when to use a basic table", you talk about it more as "when to use a table vs data list", so that's a little confusing. 

This is ones hard... @mcarrano do you think we should have seperate sections for Compact and Basic? Or should we combine into one. Also for the "when to use sections" should they focus on "table vs data list", or "compact vs basic"? (ie: for the compact table when to use, would you suggest talking about when to use a compact table vs a basic table, or would you suggest talking about when to use a table vs a data list". Keeping in mind that there is already a section later on where compact vs basic is explained....

Reply:
Wow. This is a tough question @mmenestr .  I think I see where you are headed with this. How about this idea. The sections that address List and Table usage should focus on the feature options for a list or table (not styling). Therefore use a consistent styling for all the examples. The the section called "when to use a compact table" can just focus on explaining the styling options and when to use one of the other. Does that make sense @mmenestr @doruskova ? Or am I making it more complicated?

Reply:
@mmenestr @mcarrano I've added a section for a basic data list. And I agree that there's an inconsistency between these two sections. What about to explain it from both perspectives? Or merge the Compact and Basic table/data list sections? Because the compact and basic deliver the same message from a styling perspective. But, I think it's for a deeper discussion. However, I fixed everything. This is the only thing that's missing.